auth-tokens/src/main/java/com/palantir/tokens/auth/BearerTokens.java

```diff
@@ -78,7 +78,7 @@ public static BearerToken fromPath(Path path) throws IOException {
         }
 
         LOGGER.warn("No token file found in any of the configured paths: {}.", paths);
-        return Optional.absent();
+        return Optional.empty();
```

Comment:
Yeah please, can we just kill the tokens2 package and move back to tokens? You end up converting between them _everywhere_ and it's a giant pain. It's especially galling since the only two classes most people see have no actual breaking changes, and the ones that did break have no real diamond dependency issues.